src/Http/Middleware/CsrfProtectionMiddleware.php

```diff
@@ -66,6 +66,13 @@ class CsrfProtectionMiddleware
      */
     protected $_config = [];
 
+    /**
+     * Callback for allowing to skip token check for particular request.
```

Comment:
```suggestion
     * Callback for deciding whether or not to skip the token check for particular request.
```

Reply:
Might also want to mention that returning true from this callback will skip the csrf check.